test/scripts/utils.test.js

```diff
@@ -56,8 +56,9 @@ describe('Auto Blocks', () => {
     document.head.innerHTML = metadata;
   });
 
-  beforeEach(() => {
+  beforeEach(async () => {
     sinon.stub(window.lana, 'log');
+    document.body.innerHTML = await readFile({ path: './mocks/body.html' });
```

Comment:
nit: read the body file once and set innerHTML using a const.
```suggestion
const body = await readFile({ path: './mocks/body.html' });
  // ...
  beforeEach(() => {
    sinon.stub(window.lana, 'log');
    document.body.innerHTML = body;
```